ground-control/internal/server/handlers.go

```
@@ -84,237 +53,696 @@
+	satExist, err := harbor.GetProject(r.Context(), "satellite")
 	if err != nil {
-		err = &AppError{
-			Message: err.Error(),
-			Code:    http.StatusBadRequest,
+		log.Println(err)
+		err := &AppError{
+			Message: fmt.Sprintf("Error: Checking satellite project: %v", err),
+			Code:    http.StatusBadGateway,
 		}
 		HandleAppError(w, err)
+		tx.Rollback()
```

golangci-lint (1.62.2)

Error return value of `tx.Rollback` is not checked

(errcheck)